common/src/main/java/tachyon/util/CommonUtils.java

```diff
@@ -29,6 +29,11 @@
 public final class CommonUtils {
   private static final Logger LOG = LoggerFactory.getLogger("");
 
+  /**
```

Comment:
Good catch. I missed them first time around; I added JavaDoc for listToString() and getFqdnHost().